scripts/package/fpsci_packager_generator.py

```diff
@@ -0,0 +1,125 @@
+# This script generates a packaging script based on the files used by the last successful run of the program.
+#
+# This script should be run from the root directory of a checkout out FPSci repository. 
+# Normally this means that you'd envoke it with `python scripts/package/fpsci_packager_generator.py`.
+# This script outputs a new version of `scripts/package/fpsci_packager.sh` based on the latest `data-files/log.txt`.
+# Both of those can be changed in the parameters below
+#
+# Once generated, `scripts/package/fpsci_packager.sh` should be run using bash from the root of the repository.
+# It will create a `dist/` directory and copy all of the needed files into it.
+#
+# The intention is that the contents of the `dist/` directory can then be zipped up for distribution.
+
+# Current release being built. Defaults to the git hash. Use "master", "branch name" or "tag name" for other builds.
+import subprocess
+currentRelease = subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).strip().decode()
+# currentRelease = "master"
+
+# Name of the packaging script to generate
+outputScript = 'scripts/package/fpsci_packager.sh'
+
+# Path of the log.txt file to use
+inputLog = 'data-files/log.txt'
+
+# The list of files that should always be excluded
+excludeList = ['log.txt', 'keymap.Any', 'systemconfig.Any']
+# The list of files that should be excluded for release builds, but may want to be included for specific experiment distributions
+# Use the empty list if you want to distribute an experiment
+secondaryExcludeList = ['experimentconfig.Any', 'startupconfig.Any', 'userconfig.Any', 'userstatus.Any']
```

Comment:
Eventually it would be nice to support optionally including these files through use of some sort of "experiment" packaging mode. But for now happy to have them left off/added manually to the packaged build.

Reply:
I agree. I think we should handle the experiment packaging as a separate Issue and Pull Request.